Jobsway2goMvc/Controllers/JobsController.cs

```diff
@@ -24,11 +28,14 @@ public JobsController(ApplicationDbContext context, IHttpContextAccessor httpCon
             _httpContextAccessor = httpContextAccessor;
         }
 
-        public async Task<IActionResult> Index()
+        public IActionResult Index(int? page)
         {
-            var result = await _context.Jobs.Include(j => j.Category).ToListAsync();
-            return View(result);
+            //Showing 3 jobs per page
+            var jobs = _context.Jobs.ToPagedList(page ?? 1,3);
+
```

Comment:
Create variables named `itemsPerPage` and `pageIndex` with default values (for cases when we don't provide values) and pass it as parameter to `ToPagedList(page ?? pageIndex, itemsPerPage);`